src/plugins/janus_audiobridge.c

```diff
@@ -7292,6 +7381,18 @@ static void *janus_audiobridge_handler(void *data) {
 				const char *ip = json_string_value(json_object_get(rtp, "ip"));
 				uint16_t port = json_integer_value(json_object_get(rtp, "port"));
 				janus_mutex_lock(&participant->pmutex);
+				/* rfc2833 payload type is set */
+				json_t *dtmf_pt_json =json_object_get(rtp, "dtmf_pt");
```

Comment:
Besides what Alessandro already said, all new API properties must be validated before they can be accessed. See the way "ip" and "port" are validated right now in the code, in the related parameters structure, and add the new property there too.

I also see a missing space before `json_object_get` (code style).